v0.2.8 and v0.2.9 - Module not found: Can't resolve 'fs/promises'

I tried to upgrade `@electric-sql/pglite` from 0.2.7 to 0.2.8 yesterday, and then again today to 0.2.9, with the same result.

Here is the build output:

```sh
> next build 

  ▲ Next.js 15.0.0-canary.161
  - Environments: .env

   Creating an optimized production build ...
Failed to compile.

../../node_modules/.pnpm/@electric-sql+pglite@0.2.9/node_modules/@electric-sql/pglite/dist/chunk-AZTFRW3V.js
Module not found: Can't resolve 'fs/promises'

https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
../../node_modules/.pnpm/@electric-sql+pglite@0.2.9/node_modules/@electric-sql/pglite/dist/index.js
```

I have PGlite running with Next.js 15 like a charm with 0.2.7, per https://github.com/electric-sql/pglite/issues/322#issuecomment-2372563526

--- a/.changeset/funny-candles-warn.md
+++ b/.changeset/funny-candles-warn.md
@@ -1,0 +1,5 @@
+---
+'@electric-sql/pglite': patch
+---
+
+Add `fs/promises: false` to the browser config in package.json to exclude it from browser builds.

--- a/packages/pglite/package.json
+++ b/packages/pglite/package.json
@@ -106,6 +106,7 @@
   },
   "browser": {
     "fs": false,
+    "fs/promises": false,
     "path": false,
     "url": false,
     "zlib": false,

